### PR TITLE
[Madagascar] [HW1] Created StudentMadagascar.kt

### PR DESCRIPTION
**Pull request by neolost**
As a home work 1, Created properties and functions in StudentMadagascar.kt

### CODE REVIEW

**Comment by neolost** [comment]
@booorisss Could you take a look at this PR ?

**Comment by tagirf** [comment]
test

File: app/src/main/java/com/example/kotlinplayground/StudentMadagascar.kt [hunk 1]
```diff
@@ -0,0 +1,56 @@
+package com.example.kotlinplayground
+
+open class StudentMadagascar(var lastName: String, var course: Int, var yearTaking: Int) {
```
Comment (tagirf) [inline app/src/main/java/com/example/kotlinplayground/StudentMadagascar.kt]: Should not be open

**Review by tagirf — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/kotlinplayground/StudentMadagascar.kt (hunk 1 above)
Comment (neolost) [reply app/src/main/java/com/example/kotlinplayground/StudentMadagascar.kt]: Check latest commit

**Review by tagirf — APPROVED** [review]

**Review by yegorovmax — APPROVED** [review]

**Pull request merged**